openlibrary/templates/login.html

```diff
@@ -69,5 +63,5 @@ <h1>$_("Log In")</h1>
             <br/><br/>
```

Comment:
Can we remove these <br>'s while we're at it?

Reply:
Sure

Reply:
@mekarpeles I think not a member of open library would look better in the next line :)